aptos-move/framework/aptos-framework/sources/fungible_asset/managed_fungible_metadata.move

```diff
@@ -0,0 +1,294 @@
+/// This module provides an addtional ready-to-use solution on top of `FungibleAssetMetadata` that manages the refs of
+/// mint, burn and transfer for the creator in a straightforward scheme. It offers creators to destory any refs in an
+/// on-demand manner too.
+module aptos_framework::managed_fungible_metadata {
+    use aptos_framework::fungible_asset::{Self, MintRef, TransferRef, BurnRef, FungibleAsset};
+    use aptos_framework::fungible_store;
+    use aptos_framework::object::{Self, Object, ConstructorRef};
+    use std::error;
+    use std::option::{Self, Option};
+    use std::signer;
+    use std::string::String;
+
+    /// MintRef existence error.
+    const EMINT_REF: u64 = 1;
+    /// TransferRef existence error.
+    const ETRANSFER_REF: u64 = 2;
+    /// BurnRef existence error.
+    const EBURN_REF: u64 = 3;
+    /// Not the owner.
+    const ENOT_OWNER: u64 = 4;
+    /// Refs existence errors.
+    const EMANAGED_FUNGIBLE_ASSET_REFS: u64 = 5;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Hold refs to control the minting, transfer and burning of fungible assets.
+    struct ManagingRefs has key {
+        mint: Option<MintRef>,
+        transfer: Option<TransferRef>,
+        burn: Option<BurnRef>,
```

Comment:
let's not make these optional, they should be here

Reply:
what if I want the manager to forgo some of them explicitly?